lazy/init.go

```diff
@@ -76,14 +75,15 @@ func (ini *Init) Do() (interface{}, error) {
 		panic("init is nil")
 	}
 
+	var err error
+
 	ini.init.Do(func() {
```

Comment:
Wouldn't doing a `ini.err = nil` here have the same functional effect?

Reply:
I wanted to remove the `ini.err` field since it wasn't used outside this function in order to simplify the `Init` struct a bit and reduce the risk of accidental memory sharing.